form.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="es">
+    <head>
+    <meta charset="utf-8">
+    <meta name="og:title" property="og:title" content="Ferreteria Trinidad la 14">
+    <meta name="robots" content="2">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta name="description" content="La ferretería Trinidad ofrece los servicios de cerrajero y plomero a domicilio en la ciudad de tunja, además encuentra en nuestras instalaciones todos los materiales para la construcción">
+    <title>Ferreteria Trinidad</title>
+    </head>
+    <body>
+    <header>
+     <h1>Ferreteria</h1>
+     <h2>Bienvenidos a la Ferreteria Trinidad la 14</h2>
+     <nav>
+      <ol>
+       <li>Inicio</li>
+       <li>Pituras</li>
+       <li>Herramientas</li>
+       <li>Tuberia</li>
+       <li>Accesorios</li>
+      </ol>
+     </nav>
+    </header>
+    <main>
+    <article>
+     <header>
+      <h2>Promociones:</h2>
+      <figure>
+      <img src="3-semantic-html/imagenes/F1.jpeg" width="250" height="250" alt="Productos de ferreteria"><br>
+      <figcaption>Productos de ferreteria</figcaption>
+      </figure>
+     </header>
+     <section>
+      <p>Esta ferreteria es un establecimiento comercial que se dedica  a la venta al por mayor y menor de materiales para construccion y necesidades del hogar</p>  
+      <img src="3-semantic-html/imagenes/F2.jpeg" width="400" height="200" alt="Fachada"><br>
+      <p>Realiza tus pedidos:</p>
```

Comment:
```suggestion
      <h3>Realiza tus pedidos:</h3>
```
The section needs a heading and this one is the best to meet that requirement.